cmd/snap-confine/snap-confine.rst

```diff
@@ -129,7 +129,7 @@ FILES
 
 	Description of the mount profile.
 
-`/var/lib/snapd/seccomp/profiles/*`:
+`/var/lib/snapd/seccomp/bpf/*`:
```

Comment:
I can add this, strictly speaking snap-confine does not care for the `.in` file though. What do you think about:
```
`/var/lib/snapd/seccomp/bpf/*.bpf`:

	Compiled seccomp profile program.

`/var/lib/snapd/seccomp/bpf/*.in`:

	Input for the /usr/lib/snapd/snap-seccomp profile compiler.
```

Note that we currently have both the .bpf and the .in in the same directory. The reason is mostly that we need a different place than `/var/lib/snapd/profiles`. This directory must be kept untouched or will will loose the ability to rollback to older versions. The code in `EnsureDirState` will clean all snap.$snap-name files. So in order to be able to rollback to version prior to this change we need to be able to revert to this previous place. If there is a strong desire to keep the .in and the .bpf in separate dirs, we could use something like /var/lib/snapd/seccomp/{input,bpf} ?